### PR TITLE
telephone numbers are now a list of objects with name and number

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

**Comment by marcofucci** [comment]
:+1: but can you double-check, @teneightfive ?

File: cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html [hunk 1]
```diff
@@ -40,7 +40,11 @@ <h1 class="u-pullLeft">Alternative help</h1>
       <div class="Grid">
         <div class="Grid-row cf">
           <div class="Grid-col Grid-col--1-3">{{ ::provider.address }}</div>
-          <div class="Grid-col Grid-col--1-3" ng-bind-html="::provider.helpline|escapeHtml|nl2br"></div>
+          <div class="Grid-col Grid-col--1-3" ng-If="provider.telephone_numbers">
```
Comment (teneightfive) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: should be a lowercase `if`

Comment (teneightfive) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: Can you also prefix the scope variable with `::`. It prevents it being a two-way binding and creating too many watches

File: cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html [hunk 2]
```diff
@@ -80,7 +84,11 @@ <h1 class="u-pullLeft">Alternative help</h1>
         <span class="Grid">
           <span class="Grid-row cf">
             <span class="Grid-col Grid-col--1-3">{{ ::provider.address }}</span>
-            <span class="Grid-col Grid-col--1-3" ng-bind-html="::provider.helpline|escapeHtml|nl2br"></span>
+            <span class="Grid-col Grid-col--1-3" ng-If="provider.telephone_numbers">
```
Comment (teneightfive) [inline cla_frontend/assets-src/javascripts/app/partials/case_detail.alternative_help.html]: Same here. Add `::` to `provider.telephone_numbers` and `value.name`

**Comment by teneightfive** [comment]
:+1:

**Pull request merged**